manifests/b/byCrookie/TypeCode/1.1.11/byCrookie.TypeCode.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: byCrookie.TypeCode
+PackageVersion: 1.1.11
+InstallerType: msi
+InstallModes:
+  - silent
+Installers:
+  - InstallerUrl: https://github.com/byCrookie/TypeCode/releases/download/v1.1.11/TypeCode.Wpf.Setup.msi
+    Architecture: x64
+    InstallerSha256: b413e5f0f4460a406225f1ee07d43aa71ac6148aa79f559f134960436c88f12f
```

```suggestion
    InstallerSha256: b413e5f0f4460a406225f1ee07d43aa71ac6148aa79f559f134960436c88f12f
    ProductCode: '{C7465337-21EF-45C1-A6CC-2101884FF304}'
```
ProductCode field of MSI Installers for better upgrade matching.